lib/Convos/Core/Dialog.pm

```diff
@@ -36,12 +37,15 @@ sub calculate_unread {
   );
 }
 
+sub stash { Convos::Util::_stash(stash => @_) }
```

Comment:
I'm not sure I'm happy with this name, but I think we need something like this to prevent hard coupling between plugins and objects.